docker-compose.env.sample

```diff
@@ -9,6 +9,7 @@ AUTHORISATION_SERVICE_URL=
 AUTHORISATION_SERVICE_TOKEN=
 GOOGLE_DRIVE_JSON_KEY=
 GOOGLE_GEOCODING_API_KEY=
+OS_GEOCODING_API_KEY=
```

Comment:
Can we be verbose about using Ordinance Survey instead of OS for clarity (since in most computer contexts, OS is Operating System)?